tests/utest/zap/test_zap_cli.py

```diff
@@ -0,0 +1,118 @@
+from ..helpers import RESOURCES_PATH
+from oxygen.oxygen import OxygenCLI
+from robot.running.model import TestSuite
+from unittest import TestCase
+from unittest.mock import ANY, create_autospec, Mock, patch
+
+
+import sys
+
+
+class TestOxygenZapCLI(TestCase):
+    zap_xml = str(RESOURCES_PATH / 'zap' / 'zap.xml')
+    base_args = ['oxygen', 'oxygen.zap', zap_xml]
+
+    def setUp(self):
+        self.cli = OxygenCLI()
+        self.handler = self.cli._handlers['oxygen.zap']
+
+    def test_cli(self):
+        self.assertEqual(
+            self.handler.cli(),
+            {('--accepted-risk-level',): {
+                'help': 'Set accepted risk level',
+                'type': int},
+             ('--required-confidence-level',): {
+                 'help': 'Set required confidence level',
+                 'type': int},
+             ('result_file',): {}})
+
+    @patch('oxygen.oxygen.RobotInterface')
+    @patch.object(sys, 'argv', base_args)
```

Comment:
This is one of the critical parts to understand from the test case and thus, it's very weird that it's not in the context of code, but outside of it in a decorator.

Also, needing to scroll up to see `base_args` is bit cumbersome, where it could be in the context of the code

`patch.object()` can be used as a context manager (example from mock documentation):
```python
with patch.object(ProductionClass, 'method', return_value=None) as mock_method:
    thing = ProductionClass()
    thing.method(1, 2, 3)

mock_method.assert_called_once_with(1, 2, 3)
```
So in other words, something like this would be nice:
```python
@patch('oxygen.oxygen.RobotInterface')
def test_cli_no_optional_arguments(self, mock_robot_iface):
     cmd = f'oxygen oxygen.zap {self.ZAP_XML}'  # here you can see what the actual command is
     with patch.object(sys, 'argv', cmd.split()):  
         self.cli.run()

    self.assertEquals(self.handler._config['accepted_risk_level'], 2)
    self.assertEquals(self.handler._config['required_confidence_level'], 1)